acasclient/experiment.py

```diff
@@ -86,6 +86,15 @@ def protocol_name(self) -> str:
                 label_kind="protocol name")
         return protocol_name["labelText"]
 
+    @property
+    def ls_kinds(self) -> list:
```

Comment:
@brianbolt My one issue here is that "ls_kind" is really an internal concept to ACAS, and won't be immediately understood by users of acasclient. I think calling the property `column_names` or `result_types` would be a lot clearer as to what these strings represent.

Reply:
Thanks makes sense but how about `result_kinds`?  Only because result types is really numeric, string...etc. and column names would be the combination of "result_kind" + units

Reply:
I'm good with `result_kinds`